modules/Kruskal_algorithm/include/Kruskal_algorithm.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Volkova Anastasia
+
+#ifndef MODULES_KRUSKAL_ALGORITHM_INCLUDE_KRUSKAL_ALGORITHM_H_
+#define MODULES_KRUSKAL_ALGORITHM_INCLUDE_KRUSKAL_ALGORITHM_H_
+
+#include <utility>
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+typedef std::pair<int, int> edge;
+
+class Graph{
+ private:
+    int n;
+    std::vector<std::pair<int, edge>> G;
+    std::vector<std::pair<int, edge>> MST;
+    std::vector<int> parent;
+    void make_set(int x);
+    int find_set(int i);
+    void union_set(int a, int b);
+ public:
+    Graph();
+    explicit Graph(const int n);
+    Graph(const Graph& tmp);
+    Graph& operator=(const Graph& tmp);
+    ~Graph();
```

Comment:
Поскольку внутри класса используются только контейнеры STL и целочисленные переменные, то деструктор можно сделать по умолчанию, логика нарушена не будет
```suggestion
    ~Graph() = default;
```